src/python/tensorflow_cloud/core/tests/unit/validate_test.py

```diff
@@ -16,7 +16,9 @@
 import pytest
 import unittest
 
+from mock import patch
 from tensorflow_cloud.core import machine_config
+from tensorflow_cloud.core import deploy
```

Comment:
Why are we importing deploy here? Shouldn't be required.

Reply:
good catch, removed it.